index.html

```diff
@@ -1,339 +1,300 @@
+                                    <img src="./images/css-ic.svg" alt="css"><span>CSS</span>
+                                </li>
+                            </ul>
+                        </li>
+                        <li class="topics">
+                            Frame Works
+                            <img src="./images/Icon-Chevron-Down-Larger.svg" alt="drop-down">
+                        </li>
+                        <li class="topics">
+                            Skills
+                            <img src="./images/Icon-Chevron-Down-Larger.svg" alt="drop-down">
+                        </li>
+                    </ul>  
+                </div>
+            </div>
+        </div>
     </section>
+  
 
-    <footer>
-      <section class="background-style">
-        <div class="contact-form" id="contact">
-        <h2 class="contact-title">Contact me</h2>
-        <p class="p-contact">
-          If you have an application you are interested in developing, a feature
-          that you need built or a project that needs coding. I’d love to help
-          with it
-        </p>
-        <form
-          class="contact-info"
-          action="https://formspree.io/f/mayvgkrd"
-          method="POST"
-        >
-          <input
-            type="text"
-            name="Name"
-            class="inp-form"
-            maxlength="30"
-            required
-            placeholder="Full name"
-          />
-
-          <input
-            type="email"
-            name="_replyTo"
-            class="inp-form"
-            id="email"
-            placeholder="email"
-            required
-          />
-          <textarea
-            name="message"
-            class="text-area"
-            cols="30"
-            rows="10"
-            maxlength="500"
-            placeholder="Write your message here"
-            required></textarea>
+<footer class="background-style">
+  <section class="contact-form" id="contact">
+    <h2 class="contact-title">Contact me</h2>
+    <p class="p-contact">
+      If you have an application you are interested in developing, a feature
+      that you need built or a project that needs coding. I’d love to help
+      with it
+    </p>
+    <form class="input" action="https://formspree.io/f/mayvgkrd" method="post">
+        <input type="text" class="input-item" name="Name" placeholder="full name" maxlength="30" required>
+         <input type="email" id="email" class="input-item" name="Email" placeholder="email" required>
+        <textarea name="message" cols="30" rows="10" placeholder="write your message here" maxlength="500" required></textarea>
+        <small class="error"></small>
+        <button type="submit" id="submit">Get in Touch</button>
+     </form>                                                                
+    <div class="contact-bar">
+      <img class="end-line" src="./images/Shape.png" alt=" end line icon" />
+    </div>
+  </section>
+</footer>
```

Comment:
### Your Contact form
![contact-form](https://user-images.githubusercontent.com/93039359/181797805-249a9ac9-0f10-40c8-a044-f048fe6d4a4e.png)

### Figma Contact form
![figma](https://user-images.githubusercontent.com/93039359/181797832-c1a57c34-6164-49a7-92f7-9190ed8e214d.png)

**[Optional]**
- [ ] You can compare your contact form with the Figma contact form. Please try to make your contact form look like the Figma one. It would make your project look professional and will attract the recruiters in near future :blush: